server/graphql/programs/src/queries/patient.rs

```diff
@@ -1,153 +1,16 @@
 use async_graphql::*;
-use chrono::{Local, NaiveDate};
 use graphql_core::generic_filters::{
     DateFilterInput, EqualFilterStringInput, SimpleStringFilterInput,
 };
-use graphql_core::loader::DocumentLoader;
 use graphql_core::map_filter;
 use graphql_core::pagination::PaginationInput;
 use graphql_core::{standard_graphql_error::validate_auth, ContextExt};
 use graphql_general::{EqualFilterGenderInput, GenderInput};
-use graphql_types::types::GenderType;
-use repository::{
-    DateFilter, EqualFilter, Pagination, PaginationOption, ProgramEnrolmentFilter,
-    SimpleStringFilter,
-};
+use repository::{DateFilter, EqualFilter, PaginationOption, SimpleStringFilter};
 use service::auth::{CapabilityTag, Resource, ResourceAccessRequest};
-use service::programs::patient::{
-    main_patient_doc_name, Patient, PatientFilter, PatientSort, PatientSortField,
-};
-
-use crate::types::document::DocumentNode;
-use crate::types::program_enrolment::ProgramEnrolmentNode;
-
-use super::ProgramEnrolmentFilterInput;
-
-pub struct PatientNode {
-    pub store_id: String,
-    pub patient: Patient,
-    pub allowed_ctx: Vec<String>,
-}
-
-#[Object]
-impl PatientNode {
-    pub async fn id(&self) -> &str {
-        &self.patient.name_row.id
-    }
-
-    pub async fn name(&self) -> &str {
-        &self.patient.name_row.name
-    }
-
-    pub async fn code(&self) -> &str {
-        &self.patient.name_row.code
-    }
-
-    pub async fn code_2(&self) -> &Option<String> {
-        &self.patient.name_row.national_health_number
-    }
-
-    pub async fn first_name(&self) -> Option<String> {
-        self.patient.name_row.first_name.clone()
-    }
-
-    pub async fn last_name(&self) -> Option<String> {
-        self.patient.name_row.last_name.clone()
-    }
-
-    pub async fn gender(&self) -> Option<GenderType> {
-        self.patient
-            .name_row
-            .gender
-            .as_ref()
-            .map(GenderType::from_domain)
-    }
-
-    pub async fn date_of_birth(&self) -> Option<NaiveDate> {
-        self.patient.name_row.date_of_birth.clone()
-    }
-
-    pub async fn age(&self) -> Option<i64> {
-        self.patient.name_row.date_of_birth.clone().map(|dob| {
-            let diff = Local::now().naive_utc().date().signed_duration_since(dob);
-            diff.num_days() / 365
-        })
-    }
-
-    pub async fn phone(&self) -> Option<String> {
-        self.patient.name_row.phone.clone()
-    }
-
-    pub async fn country(&self) -> Option<String> {
-        self.patient.name_row.country.clone()
-    }
-
-    pub async fn address1(&self) -> Option<String> {
-        self.patient.name_row.address1.clone()
-    }
-
-    pub async fn address2(&self) -> Option<String> {
-        self.patient.name_row.address2.clone()
-    }
-
-    pub async fn email(&self) -> Option<String> {
-        self.patient.name_row.email.clone()
-    }
+use service::programs::patient::{PatientFilter, PatientSort, PatientSortField};
 
```

Comment:
Would assume this is just moved ? (A self review comment would be helpful here, i.e.  "just moved to type with no extra changes")

Reply:
This was to have the type available in the new PatientLoader...

Reply:
> This was to have the type available in the new PatientLoader...

Yep, that's understandable, do you agree though that it's a little bit easier to review if there is a self review comment saying there is not actual changes ?